### DISCUSSION TITLE
How to skip the beginning page and directly go to the track page?

### DISCUSSION THREAD

**Discussion by Sylarair** [GMOD/jbrowse-components#2618] [Q&A] [answered]
Hi, I construct a track with JBrowse successfully. However, there is a beginning page that shows "Start a new session" which clicking the Browser button on the top menu bar (see the attached figure), and I want to skip it. Could anyone tell me what I should do?

<img width="1041" alt="Screen Shot 2021-12-29 at 4 29 20 PM" src="https://user-images.githubusercontent.com/18133362/147642321-bd50dffc-1eff-4ed3-a81f-5289f4b8c8cb.png">

**Comment by cmdcolin** [comment, marked as answer]
this is generally related to configuring a "default session". if you use `jbrowse admin-server` there is a special menu item to help, or you can go to "File->Export session", and put the session.json contents in your config.json under a key named defaultSession
